soccer/gameplay/root_play.py

```diff
@@ -2,13 +2,13 @@
 from play import Play
 import robocup
 from behavior import Behavior
-import plays.stopped
+import plays.Stopped.basic_stopped
```

Comment:
So the way I've rearranged this is that Stopped is basically a situation folder, and all the situation folders are capitalized. See the link below for the file structure under this PR. Every folder that is capitalized has an __init__.py in it and the folders that are lower-case do not.

It might not be super obvious that these changes have been made due to how github displays "name changes" but I would like people to look at this reorganization in general.

[Everyone make sure to check it out to see if we are ok with these changes](https://github.com/RoboJackets/robocup-software/tree/reAddDefense/soccer/gameplay/plays)

(legacy plays have been moved to lowercase legacy, no __init__.py, stopped.py which was in no directory has been moved to Stopped, the restarts folder has been emptied and it's contents distributed amongst other folders. Amongst other things)

Reply:
I'm not a fan of this. IMO the best solution is to stop making directory names correspond 1:1 with display names, i.e. by exporting a string from `__init__.py` for the situation folder name is.